src/box/errcode.h

```diff
@@ -115,286 +115,286 @@ struct errcode_record {
+	_(ER_NO_SUCH_SPACE, 36,			"Space does not exist", "space", STRING) \
+	_(ER_NO_SUCH_FIELD_NO, 37,		"Field does not exist", "fieldno", INT) \
+	_(ER_EXACT_FIELD_COUNT, 38,		"Tuple field count does not match space format field count", "actual", UINT, "expected", UINT) \
+	_(ER_FIELD_MISSING, 39,			"Tuple field is required by space format", "field", STRING) \
 	_(ER_WAL_IO, 40,			"Failed to write to disk") \
-	_(ER_MORE_THAN_ONE_TUPLE, 41,		"Get() doesn't support partial keys and non-unique indexes") \
-	_(ER_ACCESS_DENIED, 42,			"%s access to %s '%s' is denied for user '%s'") \
-	_(ER_CREATE_USER, 43,			"Failed to create user '%s': %s") \
-	_(ER_DROP_USER, 44,			"Failed to drop user or role '%s': %s") \
-	_(ER_NO_SUCH_USER, 45,			"User '%s' is not found") \
-	_(ER_USER_EXISTS, 46,			"User '%s' already exists") \
+	_(ER_MORE_THAN_ONE_TUPLE, 41,		"Get() does not support partial keys and non-unique indexes") \
+	_(ER_ACCESS_DENIED, 42,			"Access denied") \
+	_(ER_CREATE_USER, 43,			"Failed to create user", "user", STRING, "details", STRING) \
+	_(ER_DROP_USER, 44,			"Failed to drop user", "user", STRING, "details", STRING) \
+	_(ER_NO_SUCH_USER, 45,			"User does not exist", "user", STRING) \
+	_(ER_USER_EXISTS, 46,                   "User already exists", "user", STRING) \
 	_(ER_CREDS_MISMATCH, 47,		"User not found or supplied credentials are invalid") \
-	_(ER_UNKNOWN_REQUEST_TYPE, 48,		"Unknown request type %u") \
-	_(ER_UNKNOWN_SCHEMA_OBJECT, 49,		"Unknown object type '%s'") \
-	_(ER_CREATE_FUNCTION, 50,		"Failed to create function '%s': %s") \
-	_(ER_NO_SUCH_FUNCTION, 51,		"Function '%s' does not exist") \
-	_(ER_FUNCTION_EXISTS, 52,		"Function '%s' already exists") \
+	_(ER_UNKNOWN_REQUEST_TYPE, 48,		"Unknown request type", "request_type", UINT) \
+	_(ER_UNKNOWN_SCHEMA_OBJECT, 49,		"Unknown object type", "object_type", STRING) \
+	_(ER_CREATE_FUNCTION, 50,		"Failed to create function", "function", STRING, "details", STRING) \
+	_(ER_NO_SUCH_FUNCTION, 51,		"Function does not exist", "function", STRING) \
+	_(ER_FUNCTION_EXISTS, 52,		"Function already exists", "function", STRING) \
 	_(ER_BEFORE_REPLACE_RET, 53,		"Invalid return value of space:before_replace trigger: expected tuple or nil") \
-	_(ER_MULTISTATEMENT_TRANSACTION, 54,	"Can not perform %s in a multi-statement transaction") \
-	_(ER_TRIGGER_EXISTS, 55,		"Trigger '%s' already exists") \
-	_(ER_USER_MAX, 56,			"A limit on the total number of users has been reached: %u") \
-	_(ER_NO_SUCH_ENGINE, 57,		"Space engine '%s' does not exist") \
-	_(ER_RELOAD_CFG, 58,			"Can't set option '%s' dynamically") \
-	_(ER_CFG, 59,				"Incorrect value for option '%s': %s") \
+	_(ER_MULTISTATEMENT_TRANSACTION, 54,	"Can not perform in a multi-statement transaction", "action", STRING) \
+	_(ER_TRIGGER_EXISTS, 55,		"Trigger already exists", "trigger", STRING) \
+	_(ER_USER_MAX, 56,			"A limit on the total number of users has been reached", "limit", UINT) \
```

Comment:
Please remove the undefined article: "Limit on the total number of ..."

Reply:
Fixed everywhere.

<details>
<summary>diff</summary>

```diff
diff --git a/src/box/errcode.h b/src/box/errcode.h
index 208147db6..07a1c5aff 100644
--- a/src/box/errcode.h
+++ b/src/box/errcode.h
@@ -170,7 +170,7 @@ struct errcode_record {
        _(ER_BEFORE_REPLACE_RET, 53,            "Invalid return value of space:before_replace trigger: expected tuple or nil") \
        _(ER_MULTISTATEMENT_TRANSACTION, 54,    "Cannot perform operation in a multi-statement transaction", "operation", STRING) \
        _(ER_TRIGGER_EXISTS, 55,                "Trigger already exists", "trigger", STRING) \
-       _(ER_USER_MAX, 56,                      "A limit on the total number of users has been reached", "max", UINT) \
+       _(ER_USER_MAX, 56,                      "Limit on the total number of users has been reached", "max", UINT) \
        _(ER_NO_SUCH_ENGINE, 57,                "Space engine does not exist", "engine", STRING) \
        _(ER_RELOAD_CFG, 58,                    "Cannot set option dynamically", "option", STRING) \
        _(ER_CFG, 59,                           "Incorrect value for option", "option", STRING, "details", STRING) \
@@ -195,7 +195,7 @@ struct errcode_record {
        _(ER_TIMEOUT, 78,                       "Timeout exceeded") \
        _(ER_ACTIVE_TRANSACTION, 79,            "Operation is not permitted when there is an active transaction ") \
        _(ER_CURSOR_NO_TRANSACTION, 80,         "The transaction the cursor belongs to has ended") \
-       _(ER_CROSS_ENGINE_TRANSACTION, 81,      "A multi-statement transaction cannot use multiple storage engines") \
+       _(ER_CROSS_ENGINE_TRANSACTION, 81,      "Multi-statement transaction cannot use multiple storage engines") \
        _(ER_NO_SUCH_ROLE, 82,                  "Role does not exist", "role", STRING) \
        _(ER_ROLE_EXISTS, 83,                   "Role already exists", "role", STRING) \
        _(ER_CREATE_ROLE, 84,                   "Failed to create role", "role", STRING, "details", STRING) \
@@ -331,7 +331,7 @@ struct errcode_record {
        _(ER_UNCOMMITTED_FOREIGN_SYNC_TXNS, 214, "Found uncommitted sync transactions from other instance", "instance_id", UINT) \
        /* ER_SYNC_MASTER_MISMATCH, 215, Unused */ \
        _(ER_SYNC_QUORUM_TIMEOUT, 216,          "Quorum collection for a synchronous transaction is timed out") \
-       _(ER_SYNC_ROLLBACK, 217,                "A rollback for a synchronous transaction is received") \
+       _(ER_SYNC_ROLLBACK, 217,                "Rollback for a synchronous transaction is received") \
        _(ER_TUPLE_METADATA_IS_TOO_BIG, 218,    "Cannot create tuple: metadata size", "size", UINT) \
        _(ER_XLOG_GAP, 219,                     "Xlog gap") \
        _(ER_TOO_EARLY_SUBSCRIBE, 220,          "Cannot subscribe non-anonymous replica until join is done", "replica", STRING) \
```

</detaisl>